protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -181,8 +191,10 @@ public function actionEditManagerRole()
 
         if ($group == null) {
             throw new HttpException(404, Yii::t('AdminModule.controllers_GroupController', 'Group not found!'));
-        } else if ($value == null) {
-            throw new HttpException(400, Yii::t('AdminModule.controllers_GroupController', 'No value found!'));
+        } else {
```

Comment:
What do you think about `$group !== null && $value === null` as else if? I think its more clear that $value should be not null when $group is not null.